jetty/content.md

```diff
@@ -38,6 +38,24 @@ Webapps can be [deployed](https://www.eclipse.org/jetty/documentation/current/qu
 
 For older EOL'd images based on Jetty 7 or Jetty 8, please follow the [legacy instructions](https://wiki.eclipse.org/Jetty/Howto/Deploy_Web_Applications) on the Eclipse Wiki and deploy under `/usr/local/jetty/webapps` instead of `/var/lib/jetty/webapps`.
 
+## Configuration
+
+The configuration of the jetty server can be reported by running with the --list-config option:
```

Comment:
Should this say "Jetty server" instead of "jetty server"?

Add code formatting to "--list-config"